app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -442,6 +442,23 @@ class LuciBuildService {
     return buildBucketClient.getBuild(request);
   }
 
+  /// Get builder list whose config is pre-defined in LUCI.
+  Future<Set<String>> getAvailableBuilderSet({
```

Comment:
Do we need to handle builders that are from another repo's ci.yaml?

Reply:
This will cover all repos, as they share the same project which is `flutter`. From the buildbucket API side, it supports only on `project` level as well.